playground/directives/mixed-directive.ts

```diff
@@ -0,0 +1,11 @@
+import type { DirectiveBinding } from 'vue'
+
+export function NamedMixedDirective(el: HTMLElement, binding: DirectiveBinding) {
```

Comment:
I wonder if we should ask users to follow the `v` leading convention? This way it can go more explicit and easier to detect and follow the same as in Vue SFC: https://vuejs.org/guide/reusability/custom-directives#introduction

Reply:
uhmm:
- vuetify: https://github.com/vuetifyjs/vuetify/tree/master/packages/vuetify/src/directives (check any folder)
- varlet: https://github.com/varletjs/varlet/blob/9426068d339c195c4bd53fd1288d15c3c2a5b0b8/packages/varlet-ui/src/ripple/index.ts#L213

Reply:
both using named and/or default exports without `v` prefix

Reply:
In fact, it shouldn't: if the named directive starts with `vXXX`, then we need to remove the `v` prefix to match hte `resolveDirective` name, Vue will remove the `v-` prefix from the template.

Reply:
I'm changing the logic to remove the prefix: the `vXXX`  prefix should be only in the dts for the directive (Volar)